server/src/main/java/com/hedera/block/server/health/HealthServiceImpl.java

```diff
@@ -21,8 +21,11 @@
 import io.helidon.webserver.http.HttpRules;
 import io.helidon.webserver.http.ServerRequest;
 import io.helidon.webserver.http.ServerResponse;
+import javax.inject.Inject;
+import javax.inject.Singleton;
 
 /** Provides implementation for the health endpoints of the server. */
+@Singleton
 public class HealthServiceImpl implements HealthService {
 
     private static final String LIVENESS_PATH = "/liveness";
```

Comment:
I agree with you but since doing those changes implies some other changes that are out of the scope of this PR, I will open another PR that can improve on those in another more focused PR just for that.
https://github.com/hashgraph/hedera-block-node/issues/119